kinetic/distribution.yaml

```diff
@@ -758,7 +758,7 @@ repositories:
       tags:
         release: release/kinetic/{package}/{version}
       url: https://github.com/asmodehn/catkin_pip-release.git
```

Comment:
@asmodehn Just checking: this is released from a different organization than the one you released [in lunar](https://github.com/ros/rosdistro/pull/15762/files#diff-181c75201c0e263aff414d0b7d50f235R248). The source repository is also different. Are they both used for development and release or do the urls need to be updated to point to the same org ?